### ISSUE TITLE
Assertion in ThreadPredicates::updateBitSet

### ISSUE THREAD

**Issue by naoyam** [csarofeen/pytorch#111]
## ❓ Questions and Help

Why is this assertion required?

https://github.com/csarofeen/pytorch/blob/ec06f88dc64213c742ca25f6898450cda643bc7b/torch/csrc/jit/codegen/cuda/lower_thread_predicate.cpp#L141

In the softmax test I'm currently working on, it seems the assertion needs to be disabled:

https://github.com/csarofeen/pytorch/pull/100/files#diff-32ba7310cdb76c3dde2adab830e365f3R140-R149

**Comment by naoyam** [comment]
Closed by #118.
